### DISCUSSION TITLE
Create issue with attachements

### DISCUSSION THREAD

**Discussion by HiranChaudhuri** [hub4j/github-api#1461] [Q&A] [answered]
I like this fine API - it allowed me to much faster implement some functionality. So I have an application that can report stange behaviour directly as an issue. The user would just enter what he was doing/intended to do in a form field and upon clicking ok the issue is created.

Now I'd like to add a screenshot and the logfile, but somehow I am lost. What I can do is add a link into the issue text - but where does the content go? Do I have to create GHGist or GHBlob? 

How do I correctly upload and reference image and text data via the API?

**Comment by gsmet** [comment, marked as answer]
AFAICS it's not exposed in the GitHub REST API thus why you can't do it with the Hub4j GitHub API.

I couldn't find anything in the doc and it looks like others are complaining about it too: https://github.com/isaacs/github/issues/1133#issuecomment-860554483 .

**Comment by HiranChaudhuri** [comment]
Thank you for the comment. Indeed it seems that usecase is simply not wanted - but more as a 'security by obscurity' layer.
GitHub could make an announcement somewhere, the information could be part of the documentation.

Even better would be to make this a payable feature. After all it is a good feature, the API functionality is almost there and it could bring paying customers to GitHub. But I am not their product manager.

As an effect, i will search for other locations on the internet to store the file that are required only until an issue is analyzed.

**Reply by bitwiseman** [reply]
Note: This project is separate from GitHub.  If you have feedback for GitHub, it needs to be delivered via other channels.
